sympy/physics/_biomechanics/musculotendon.py

```diff
@@ -100,20 +107,47 @@ class MusculotendonDeGroote2016(ForceActuator, _NamedMixin):
         The pathway that the actuator follows. This must be an instance of a
         concrete subclass of ``PathwayBase``, e.g. ``LinearPathway``.
     activation_dynamics : ActivationBase
-
+        The activation dynamics that will be modeled within the musculotendon.
+        This must be an instance of a concrete subclass of ``ActivationBase``,
+        e.g. ``FirstOrderActivationDeGroote2016``.
     musculotendon_dynamics : MusculotendonFormulation | int
-
-    tendon_slack_length :
-
-    peak_isometric_force :
-
-    optimal_fiber_length :
-
-    maximal_fiber_velocity :
-
-    optimal_pennation_angle :
-
-    fiber_damping_coefficient :
+        The formulation of musculotendon dynamics that should be used
+        internally, i.e. rigid or elastic tendon model, the choice of
+        musculotendon state etc. This must be a member of the integer
+        enumeration ``MusculotendonFormulation`` or an integer that can be cast
+        to a member. The default is ``MusculotendonFormulation.RIGID_TENDON``
+        (or ``0``), which corresponds to a rigid tendon formulation.
```

Comment:
I've been writing/reading Python for many years and never saw anyone use an enum until here. This doc string is a little vague. I think it would be clearer just to say what exactly to set it to and the attributes on MusculotendonFormulation are the clearest (because noone will associate a integer with a formulation). Something like:

Set to MusculotendonFormation.RIGID_TENDON for ..., set to MusculotendonFormulation.SOMETHING for ..., etc.